Patterns/Korean/Korean-NumbersWithUnit.yaml

```diff
@@ -0,0 +1,781 @@
+    Vanuatu vatu: 바투 #瓦努阿图瓦图
+#Vietnamese dong
+    Vietnamese dong: 베트남 동 #越南盾
+    Indonesian rupiah: 루피아 #印度尼西亚盾
+    Netherlands guilder: 네덜란드 휠던|네덜란드 길더 #荷兰盾|荷属安的列斯盾|列斯盾
+    Surinam florin: 苏里南盾
+    Guilder: 盾
+#Zambian kwacha
+    Zambian kwacha: 赞比亚克瓦查
+#Dirham
+    Moroccan dirham: 모로코 디르함 #摩洛哥迪拉姆
+    United Arab Emirates dirham: 아랍에미리트 디르함 #阿联酋迪拉姆
+#Manat
+    Azerbaijani manat: 아제르바이잔 마나트 #阿塞拜疆马纳特
+    Turkmenistan manat: 투르크메니스탄 마나트 #土库曼马纳特
+    Manat: 마나트 #马纳特
+#Shilling
+    Somali shilling: 소말리아 실링 #索马里先令|索马利先令
+    Somaliland shilling: 소말릴랜드 실링 #索马里兰先令
+    Tanzanian shilling: 탄자니아 실링 #坦桑尼亚先令
+    Ugandan shilling: 우간다 실링 #乌干达先令
+#Leu
+    Romanian leu: 루마니아 레우 #罗马尼亚列伊
+    Moldovan leu: 몰도바 레우 #摩尔多瓦列伊
+    Leu: 레우 #列伊
+    Ban: 巴尼
+#Rupee
+    Nepalese rupee: 네팔 루피 #尼泊尔卢比
+    Pakistani rupee: 파키스탄 루피 #巴基斯坦卢比
+    Indian rupee: 인디안 루피 #印度卢比
+    Seychellois rupee: 세이셜 루피 #塞舌尔卢比
+    Mauritian rupee: 모리셔스 루피 #毛里求斯卢比
+    Maldivian rufiyaa: 루피야 #马尔代夫卢比
+    Sri Lankan rupee: 스리랑카 루피 #斯里兰卡卢比
+    Rupee: 루피 #卢比
+#Krone
+#Czech koruna
+    Czech koruna: 체코 코루나 #捷克克朗
+    Danish krone: 덴마크 크로네 #丹麦克朗|丹麦克郎
+    Norwegian krone: 노르웨이 크로네 #挪威克朗
+    Faroese króna: 페로 제도 크로나 #https://ko.wikipedia.org/wiki/%ED%8E%98%EB%A1%9C_%EC%A0%9C%EB%8F%84_%ED%81%AC%EB%A1%9C%EB%82%98#法罗克朗
+    Icelandic króna: 아이슬란드 크로나 #https://ko.wikipedia.org/wiki/%EC%95%84%EC%9D%B4%EC%8A%AC%EB%9E%80%EB%93%9C_%ED%81%AC%EB%A1%9C%EB%82%98#冰岛克朗
+    Swedish krona: 스웨덴 크로나 #瑞典克朗
+    Krone: 克朗
+    Øre: 奥依拉|奥拉|埃利
+#Franc
+    West African CFA franc: 非共体法郎
+    Central African CFA franc: 中非法郎|中非金融合作法郎
+    Comorian franc: 科摩罗法郎
+    Congolese franc: 콩고 프랑 #刚果法郎
+    Burundian franc: 브룬디 프랑 #布隆迪法郎
+    Djiboutian franc: 지투비 프랑 #吉布提法郎
+    CFP franc: CFP 프랑 #太平洋法郎
+    Guinean franc: 기나아 프랑 #几内亚法郎
+    Swiss franc: 스위스 프랑 #瑞士法郎
+    Rwandan franc: 르완다 프랑 #卢旺达法郎
+    Belgian franc: 比利时法郎
+    Rappen: 瑞士分|瑞士生丁
+    Franc: 프랑
+    Centime: 生丁|仙士
+#Ruble
+    Russian ruble: 러시아 루블 #俄国卢布|俄罗斯卢布
+    Transnistrian ruble: 德涅斯特卢布
+    Belarusian ruble: 벨라루시안 루블 #白俄罗斯卢布
+    Kopek: 戈比
+    Ruble: 루블 #卢布
+#Dinar
+    Algerian dinar: 알제르 디나르 #阿尔及利亚第纳尔
+    Bahraini dinar: 바레인 디나르 #巴林第纳尔
+    Iraqi dinar: 이라크 디나르 #伊拉克第纳尔
+    Jordanian dinar: 요르단 디나르 #约旦第纳尔
+    Kuwaiti dinar: 쿠웨이트 디나르 #科威特第纳尔|科威特币
+    Libyan dinar: 리비야 디나르 #利比亚第纳尔
+    Serbian dinar: 세르비아 디나르 #塞尔维亚第纳尔|塞尔维亚币
+    Tunisian dinar: 튀니지 디나르 #突尼斯第纳尔
+    Dinar: 디나르 #第纳尔
+    Fils: 费尔
+    Para: 帕拉
+    Millime: 米利姆
+#Peso
+    Argentine peso: 아르헨티나 페소 #阿根廷比索
+    Chilean peso: 칠레니아 페소 #智利比索
+    Colombian peso: 콜롬비안 페소 #哥伦比亚比索
+    Cuban peso: 쿠반 페소 #古巴比索
+    Dominican peso: 도미니칸 페소 #多米尼加比索
+    Mexican peso: 멕시코 페소 #墨西哥比索
+    Philippine peso: 필리핀 페소 #菲律宾比索
+    Uruguayan peso: 우루과이 페소 #乌拉圭比索
+    Peso: 페소 #比索
+    Centavo: 仙|菲辅币
+#Pound
+    Alderney pound: 奥尔德尼镑
+    British pound: 영국 파운드 #英镑
+    Guernsey pound: 건지섬 파운드 #https://ko.wikipedia.org/wiki/%EA%B1%B4%EC%A7%80%EC%84%AC_%ED%8C%8C%EC%9A%B4%EB%93%9C #根西镑
```

Comment:
What do these commented out cases at the end mean?

Reply:
removed